src/js/jsx/PanelSet.jsx

```diff
@@ -204,8 +205,10 @@ define(function (require, exports, module) {
                 modifierStore = flux.store("modifier"),
                 components = uiStore.components,
                 modifierState = modifierStore.getState(),
+                currentlyVisible = "",
                 nextState = {};
                 
+
```

Comment:
Also, no double newlines. I'm not sure why the CI isn't catching this. (Any ideas @shaoshing?)